ccm

```diff
@@ -92,6 +95,9 @@ else:
     cmd = sys.argv[2]
     cmd_args = [node] + sys.argv[3:]
 
+if not os.getenv('CASSANDRA_HOME') is None:
```

Comment:
Could you rephrase this as `if os.getenv('CASSANDRA_HOME') is not None`, please? PEP-8 specifies that form is preferable to prefixing with `if not`